ReactAndroid/build.gradle

```diff
@@ -449,55 +449,4 @@ afterEvaluate {
     // before configureNdkBuildDebug could be executed.
     configureNdkBuildDebug.dependsOn(preBuild)
     configureNdkBuildRelease.dependsOn(preBuild)
-
-    publishing {
```

Comment:
I _think_ these changes will have to go... not sure if we actually published or not. This code was breaking Gradle for me, apparently involving some missing binaries. But it did allow me to build the app locally.